terraform_files/baremetal/main.tf

```diff
@@ -83,7 +83,8 @@ module "masters" {
   cpu_mode          = var.master_cpu_mode
   cluster_domain    = var.cluster_domain
   vtpm2             = var.master_vtpm2
-
+  boot_devices      = var.master_boot_devices
```

Comment:
all edits have been made to the ``baremetal`` template, meaning not supporting the ``none`` (platform) and ``infra-env`` (all sorts of late-binding) templates
it's fine by me (and I think it will be easier to first concentrate on the "regular" case), just wanted to make sure we're all on the same page